src/Symfony/Component/Validator/Tests/Constraints/VideoValidatorTest.php

```diff
@@ -0,0 +1,472 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\Video;
+use Symfony\Component\Validator\Constraints\VideoValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+/**
+ * @author Kev <https://github.com/symfonyaml>
```

Comment:
missing ` * @requires extension fileinfo`

Reply:
done, thank you for the suggestion